src/site/docs/synonyms/assets/es6-samples.xml

```diff
@@ -0,0 +1,1038 @@
+<?xml version="1.0"?>
+<synonyms>
+<theme id="theme-getting-started">
+  <title>Getting started</title>
+  <synonym id="syn-code-embedding">
+    <title>Code embedding</title>
+    <code language="javascript"><![CDATA[
+<script src='program.js'></script>
+]]></code>
+  </synonym>
+  <synonym id="syn-entry-point">
+    <title>Entry point</title>
+    <code language="javascript"><![CDATA[
+// Not required.
+function main() {
+  // To be used as the entry point, but it must be
+  // called manually.
+}
+
+main();
+
+// Sometimes the entry point is written as an
+// anonymous function
+(function(){
+  // Code to be run automatically on execution
+})();
+]]></code>
+  </synonym>
+  <synonym id="syn-detect-language-unavailability">
+    <title>Detect language unavailability</title>
+    <code language="javascript"><![CDATA[
+<noscript>
+  Your browser doesn’t support JavaScript.
+</noscript>
+]]></code>
+  </synonym>
+  <synonym id="syn-printing-to-the-console">
+    <title>Printing to the console</title>
+    <code language="javascript"><![CDATA[
+console.log('Level completed.');
+]]></code>
+  </synonym>
+  <synonym id="syn-modal-alerts">
+    <title>Modal alerts</title>
+    <code language="javascript"><![CDATA[
+alert('Clicked on the button.');
+]]></code>
+  </synonym>
+</theme>
+<theme id="theme-code-modularity">
+  <title>Code modularity</title>
+  <synonym id="syn-define-a-library">
+    <title>Define a library</title>
+    <code language="javascript"><![CDATA[
+//------ animal.js -------
+export class Dog {
+  noise() => 'BARK!';
+}
+]]></code>
+  </synonym>
+  <synonym id="syn-use-a-library">
+    <title>Use a library</title>
+    <code language="javascript"><![CDATA[
+import {Dog} from 'animal';
+var fido = new Dog();
+
+// prefixes are supported to avoid namespace collisions
+import * as pets from 'animal';
+var fido = new pets.Dog();
+]]></code>
+  </synonym>
+</theme>
+<theme id="theme-variables">
+  <title>Variables</title>
+  <synonym id="syn-create--assign-value">
+    <title>Create + assign value</title>
+    <code language="javascript"><![CDATA[
+var myName = 'Aaron';
+]]></code>
+  </synonym>
+  <synonym id="syn-default-value">
+    <title>Default value</title>
+    <code language="javascript"><![CDATA[
+var myName;
+// == undefined
+
+let myName;
+// == undefined
+]]></code>
+  </synonym>
+  <synonym id="syn-hoisting">
+    <title>Hoisting</title>
+    <code language="javascript"><![CDATA[
+// EcmaScript6 "hoists" variables to the top of
```

Comment:
it might be more useful to demonstrate `let` here. It will give a similar error to Dart:
```js
function printName() {
  print(`Hello, ${name}`); // Uncaught ReferenceError: name is not defined
  let name = 'Bob';
}
```